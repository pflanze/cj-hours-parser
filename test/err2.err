Error: can't process file "test/err2.txt"

Caused by:
    0: can't parse line 2, "foo 2"
    1: can't parse as a processing instruction: "foo 2"
    2: does not start with `\w+\s*=` nor `set \w+ `
