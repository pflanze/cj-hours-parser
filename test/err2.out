item=ProcessingInstruction { key: "foo", value: "1" }
    =ProcessingInstruction { key: "foo", value: "1" }
